docs/changelog/121048.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 121048
+summary: Inference Update is PUT
```

Comment:
nit: Maybe: `Updating Inference Update API documentation to have the correct PUT method`